lib/browser/api/web-contents.js

```diff
@@ -326,6 +326,17 @@ WebContents.prototype._init = function () {
     }
   })
 
+  this.on('-ipc-invoke', function (event, channel, args) {
+    event.reply = (result) => event.sendReply({ result })
+    event.throw = (error) => event.sendReply({ error: error.toString() })
```

Comment:
use `serialize` from `lib/common/error-utils.js`?

Reply:
discussion below; resolving this comment.